src/react-apps/applications/shared/src/file-editor/FileEditor.tsx

```diff
@@ -141,13 +141,13 @@ class FileEditor extends React.Component<IFileEditorProvidedProps, IFileEditorSt
     this.loadFileContent(fileName);
   }
 
-  public saveFile = (e: any) => {
+  public saveFile = () => {
     const altinnWindow: IAltinnWindow = window as IAltinnWindow;
     const { org, service} = altinnWindow;
     const servicePath = `${org}/${service}`;
     const postUrl = `${altinnWindow.location.origin}/designer/${servicePath}/ServiceDevelopment` +
     `/SaveServiceFile?fileEditorMode=${this.props.mode}&fileName=${this.state.selectedFile}`;
-    post(postUrl, this.state.value, {headers: {'Content-type': 'text/plain;charset=utf-8'}}).then((response) => {
+    post(postUrl, this.state.value, {headers: {'Content-type': 'text/plain;charset=utf-8'}}).then(() => {
```

Comment:
I se that the post method was added before, but sine post returns a promise it might be usefull to add a _isMounted to your component so that when the promise returns the code will only perform some task if the component is still mounted.
Take a look at cloneServices in dashboard on how I have done this earlier

Reply:
Added state.mounted to resolve this